Services/CO.CDP.Forms.WebApi/Api/Forms.cs

```diff
@@ -1,27 +1,78 @@
+using CO.CDP.Forms.WebApi.Model;
 using CO.CDP.Swashbuckle.Security;
 using DotSwashbuckle.AspNetCore.SwaggerGen;
+using Microsoft.AspNetCore.Mvc;
 using Microsoft.OpenApi.Models;
 
-namespace CO.CDP.Forms.WebApi.Api
+namespace CO.CDP.Forms.WebApi.Api;
+
+public static class EndpointExtensions
 {
-    public static class EndpointExtensions
-    {
-        public static void UseFormsEndpoints(this WebApplication app)
+    private static Dictionary<Guid, Form> _forms = Enumerable.Range(1, 5)
+        .Select(_ => Guid.NewGuid())
+        .ToDictionary(id => id, id => new Form
         {
-        }
-    }
+            Id = id,
+            Name = $"Financial Information Requirement {id}",
+            Version = "1.0",
+            IsRequired = true,
+            Type = FormType.Standard,
+            Scope = FormScope.SupplierInformation,
+            Sections = new List<FormSection>
+            {
+                new FormSection
+                {
+                    Id = Guid.NewGuid(),
+                    Title = "Financial Information",
+                    AllowsMultipleAnswerSets = false,
+                    Questions = new List<FormQuestion>
+                    {
+                        new FormQuestion
+                        {
+                            Id = Guid.NewGuid(),
+                            Title = "Upload accounts or statements for your 2 most recent financial years.",
+                            Description = "If you do not have 2 years, you can upload your most recent financial year. You will need to enter the financial year end date.",
+                            Type = FormQuestionType.FileUpload,
+                            IsRequired = true,
+                            Options = new FormQuestionOptions()
+                        }
+                    },
+                    CreatedOn = DateTimeOffset.UtcNow,
+                    UpdatedOn = DateTimeOffset.UtcNow
+                }
+            },
+            CreatedOn = DateTimeOffset.UtcNow,
+            UpdatedOn = DateTimeOffset.UtcNow
+        });
 
-    public static class ApiExtensions
+    public static void UseFormsEndpoints(this WebApplication app)
     {
-        public static void DocumentFormsApi(this SwaggerGenOptions options)
-        {
-            options.SwaggerDoc("v1", new OpenApiInfo
+         app.MapGet("/forms/sample",
```

Comment:
The endpoint should return questions for the section. We agreed on this path: `/forms/{formId}/sections/{sectionId}/questions` .